internal/app/singularity/cache_clean_linux.go

```diff
@@ -187,8 +186,8 @@ func CleanSingularityCache(cleanAll bool, cacheCleanTypes []string, cacheName st
 			return err
 		}
 		if !foundMatch {
-			sylog.Warningf("No cache found with given name: %v", cacheName)
-			os.Exit(0)
+			sylog.Warningf("No cache found with given name: %s", cacheName)
+			return nil
```

Comment:
I dont think the `return nil` is necessary... Because it will `return nil` right after the `if !foundMatch {...}`.

Reply:
Good catch. Fixed.